dss/api/search.py

```diff
@@ -63,14 +67,23 @@ def post(json_request_body: dict, replica: str, per_page: int, _scroll_id: typin
 
         # TODO: (tsmith12) allow users to retrieve previous search results
         _scroll_id = page['_scroll_id']
-        result_list = [{
-            'bundle_id': hit['_id'],
-            'bundle_url': _build_bundle_url(hit, replica),
-            'search_score': hit['_score']
-        } for hit in page['hits']['hits']]
+
+        result_list = [typing.Any]
```

Comment:
Whoa, I did not know you could do this.

Although, it could be more specific, i.e., `result_list = []  # type: typing.Sequence[dict]`

Reply:
I figured out i could do that after it gave me errors for not defining the list type. I'll change it your your way.